libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFWKWebViewEngine/SFWKWebViewDelegate.m

```diff
@@ -195,6 +184,10 @@ - (void) webView:(WKWebView *) webView decidePolicyForNavigationAction:(WKNaviga
     }
 }
 
+- (void) webView:(WKWebView *) webView didCommitNavigation:(WKNavigation *) navigation {
```

Comment:
This was the key to all the problems. ```WKNavigationDelegate``` has more callbacks than I care for, but this was causing ```gap://ready``` to not fire, thereby leaving the tests in a limbo.